spec/factories/tasks.rb

```diff
@@ -1,9 +1,15 @@
 FactoryBot.define do
   factory :task do
+    association :project
     title { 'Task' }
     status { rand(2) }
     from = Date.parse("2019/08/01")
     to   = Date.parse("2019/12/31")
     deadline { Random.rand(from..to) }
```

Comment:
細かいですが`association`の定義は各カラムの値の下に書く形が良いかと思います。
```suggestion
    title { 'Task' }
    status { rand(2) }
    from = Date.parse("2019/08/01")
    to   = Date.parse("2019/12/31")
    deadline { Random.rand(from..to) }
    
    association :project
```

Reply:
modelのクラス定義の中では、associationを上に書くことが通常だと思います。(validationなどと比べて)
今回はfactorybotのカラムとassociationなので、単純比較できないかもしれませんが、何か理由はありますか？
[216024c](https://github.com/MocoAristo/rspec_app_exam/pull/1/commits/216024c12bcf9578afbcf3abf342d5f187cf8172)で修正してます。

Reply:
公式リファレンスを確認した限りでは下に書いているような気もしますが、どちらが良いか慣例など分かったら改めて連絡致します！
https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#rails-preloaders-and-rspec